docs/01-readme.md

```diff
@@ -52,7 +52,26 @@ make package
 sudo apt install *.deb
 ```
 
-
+#### NixOS (overlay in configuration.nix)
+```nix
```

Comment:
Just out of curiosity, does your markdown formatter have css class for 'nix' typed code ?

Reply:
As you can see, text on github has syntax highlightning
![image](https://user-images.githubusercontent.com/71175207/109429275-d5be5800-79f2-11eb-80d3-b41fe020e1c3.png)